lib/ansible/modules/cloud/amazon/emr_cluster_facts.py

```diff
@@ -0,0 +1,317 @@
+              returned: always
+              type: string
+              sample: 'sg-a7bc966g'
+            emr_managed_slave_security_group:
+              description: The identifier of the Amazon EC2 security group for the slave nodes
+              returned: always
+              type: string
+              sample: 'sg-5cc9265d'
+            service_access_security_group:
+              description: The identifier of the Amazon EC2 security group for the Amazon EMR service to access clusters in VPC private subnets
+              returned: always
+              type: string
+              sample: 'sg-5cc9265d'
+            additional_master_security_groups:
+              description: A list of additional Amazon EC2 security group IDs for the master node
+              returned: always
+              type: string
+              sample: 'sg-5cc9265d'
+            additional_slave_security_groups:
+              description: A list of additional Amazon EC2 security group IDs for the slave nodes
+              returned: always
+              type: string
+              sample: 'sg-5cc9265d'
+        instance_collection_type:
+          description: The instance group configuration of the cluster
+          returned: always
+          type: string
+          sample: 'INSTANCE_GROUP'
+        log_uri:
+          description: The path to the Amazon S3 location where logs for this cluster are stored
+          returned: always
+          type: string
+          sample: 's3n://rando-bucket/tmp/7346763/66338210/logs/'
+        requested_ami_version:
+          description: The AMI version requested for this cluster
+          returned: always
+          type: string
+          sample: '2.3.6'
+        running_ami_version:
+          description: The AMI version running on this cluster
+          returned: always
+          type: string
+          sample: '2.3.6'
+        release_label:
+          description: The release label for the Amazon EMR release
+          returned: always
+          type: string
+        auto_terminate:
+          description: Specifies whether the cluster should terminate after completing all steps
+          returned: always
+          type: bool
+          sample: 'true'
+        termination_protected:
+          description: Indicates whether Amazon EMR will lock the cluster to prevent the EC2 instances from being terminated
+          returned: always
+          type: bool
+          sample: 'true'
+        visible_to_all_users:
+          description: Indicates whether the cluster is visible to all IAM users of the AWS account associated with the cluster
+          returned: always
+          type: bool
+          sample: 'true'
+        applications:
+          description: The applications installed on this cluster
+          returned: always
+          type: list
+          sample: [{"name": "hadoop", "version": "1.0.3"}]
+        tags:
```

Comment:
This should be flattened to {'name': 'myAwesomeCluster} like other AWS modules. You can use boto3_tag_list_to_ansible_dict to make the transformation.